options/options.go

```diff
@@ -269,6 +273,30 @@ type TerragruntOptions struct {
 
 	// Disable listing of dependent modules in render json output
 	JsonDisableDependentModules bool
+
+	// Enables provider cache using the built-in private registry.
```

Comment:
OK, I'm going to stop fixing these on a one off basis, and just give a more general comment:

Let's standardize on "Terragrunt Provider Cache" everywhere. Not just "provider cache" or "terragrunt cache," but the full "Terragrunt Provider Cache" to make it clear this is a TG specific feature.

Reply:
Thanks for all these suggestions! I have replaced the rest, I hope I didn’t miss anything.